src/ui/simulator/windows/options/optimization/optimization.cpp

```diff
@@ -308,6 +323,52 @@ Optimization::Optimization(wxWindow* parent) :
         pBtnExportMPS = button;
     }
 
+    // Adequacy patch
+    {
+        label = Component::CreateLabel(this, wxT("Adequacy patch"));
+        button = new Component::Button(this, wxT("true"), "images/16x16/light_green.png");
+        button->SetBackgroundColour(bgColor);
+        button->menu(true);
+        onPopup.bind(this,
+                     &Optimization::onPopupMenuSpecify,
+                     PopupInfo(study.parameters.include.adequacyPatch, wxT("true")));
+        button->onPopupMenu(onPopup);
+        s->Add(label, 0, wxRIGHT | wxALIGN_RIGHT | wxALIGN_CENTER_VERTICAL);
+        s->Add(button, 0, wxLEFT | wxALIGN_LEFT | wxALIGN_CENTER_VERTICAL);
+        pBtnAdequacyPatch = button;
```

Comment:
Can you elaborate on this point ? How would that window be accessed ? Is there a piece of code we can look at with similar behavior ?

Reply:
I suggest that a new window be created, that could be accessed from the menu "Configure". It would improve the ease of use for this new feature by making all the info accessible from one place.